File: cfgs/java_cfgs_foo/COUNT_NEGATIVE_NUMBERS_IN_A_COLUMN_WISE_ROW_WISE_SORTED_MATRIX.java
Nodes
1;special;;;
2;statement;int count = 0;14;
3;statement;int i = 0;15;
4;conditional;foo_1 < foo_2;16;
5;statement;int j = 0;18;
6;conditional;foo_1 < foo_2;19;
7;conditional;foo_1[foo_2][foo_3] < 0;21;
8;statement;count += 1;21;
9;statement;break;22;
10;statement;j++;20;
11;statement;i++;17;
12;statement;return foo_1;25;
13;exit;;;
Edges
0;2;
1;3;
2;4;
3;5;12;
4;6;
5;7;11;
6;8;9;
7;10;
8;11;
9;6;
10;4;
11;13;
12;

File: cfgs/pythoncfgs/COUNT_NEGATIVE_NUMBERS_IN_A_COLUMN_WISE_ROW_WISE_SORTED_MATRIX.py
Nodes
1;statement;foo_1 = 0;8;
2;conditional;fun_1(foo_1);9;
3;statement;return foo_1;15;
4;conditional;fun_1(foo_1);10;
5;conditional;foo_1[foo_2][foo_3] < 0;11;
6;statement;foo_1 += 1;12;
7;exit;;
Edges
0;2;
1;3;4;
2;7;
3;2;5;
4;2;6;
5;4;
6;